src/test/java/org/dependencytrack/resources/v1/ComponentResourceTest.java

```diff
@@ -506,13 +506,15 @@ public void createComponentTest() {
         component.setProject(project);
         component.setName("My Component");
         component.setVersion("1.0");
+        component.setAuthor("SampleAuthor");
         Response response = jersey.target(V1_COMPONENT + "/project/" + project.getUuid().toString()).request()
                 .header(X_API_KEY, apiKey)
                 .put(Entity.entity(component, MediaType.APPLICATION_JSON));
         Assert.assertEquals(201, response.getStatus(), 0);
         JsonObject json = parseJsonObject(response);
         Assert.assertNotNull(json);
         Assert.assertEquals("My Component", json.getString("name"));
+        Assert.assertEquals("SampleAuthor" ,json.getJsonArray("authors").getJsonObject(0).getString("name"));
```

Comment:
Can you add an assertion here that `author` is also set, according to the `Component#getAuthor` logic?

Reply:
Since that endpoint just returns the `Component` object  as json, shouldn't the `author` value not be present, since it is not a value in the `Component` class?

Reply:
As you said earlier, removing that field would be a breaking change from the REST API's perspective. Internally, we don't need the legacy `getAuthor`, but users who currently rely on that field will still need it.

So I think it should be included in the response.

Reply:
I see what you mean. I'll work on this change soon